packages/tempus/src/Tempus_RKButcherTableau.hpp

```diff
@@ -240,7 +247,9 @@ class RKButcherTableau :
     bool isImplicit_;
     bool isDIRK_;
     bool isEmbedded_;
+    bool isTVD_ = false;
     Teuchos::SerialDenseVector<int,Scalar> bstar_;
+    Scalar tvdCoeff_;
```

Comment:
We should probably just have isTVD_ and tvdCoeff_ stored in a single location.  I would put it here, and remove it in StepperDIRK and StepperExplicitRK.

Reply:
ok.